tools/log-collector/log_collector.go

```diff
@@ -361,7 +362,14 @@ func (l *LogCollector) filterResourceObjects(resourcePath string,
 		}
 	}
 
-	objects = filterTvkResourcesByLabel(allObjects)
+	if nonLabeledResources.Has(resource.Kind) {
+		objects.Items = append(objects.Items, allObjects.Items...)
+	} else if clusteredResources.Has(resource.Kind) {
+		objects.Items = append(objects.Items, allObjects.Items...)
+	} else {
+		filterTvkResourcesByLabel(&allObjects)
+		return allObjects, nil
+	}
 	return objects, nil
```

Comment:
return `allObjects` here instead of  `objects`

Reply:
resolved